### DISCUSSION TITLE
[hsakmt-roct] Do tests need to be run?

### DISCUSSION THREAD

**Discussion by expenses** [rocm-arch/rocm-arch#909] [General]
These tests seem to take a long time:

https://github.com/rocm-arch/rocm-arch/blob/d396f553a1362e0c084c1ffe65150b0bb3a19ce6/hsakmt-roct/PKGBUILD#L52

Do they absolutely need to be run?

**Comment by tpkessler** [comment]
Yes, they indeed take a while. But they are needed to check if everything with `hsakmt-roct` (a fundamental package) works correctly on your system. If you don't want to build the tests, pass `--nocheck` to `makepkg`.
